pages/about.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <CContainer>
+    <h1>About</h1>
+    <h2>The DAEDALUS Explore dashboard</h2>
+    <p>
+      DAEDALUS Explore is an interactive dashboard tool that allows users to
+      simulate the health, education and economic impacts of hypothetical future
+      pandemic scenarios for 7 pandemics of varying severity in over 60
+      countries. The 7 pandemics are informed by the characteristics of
+      pathogens that have caused historic pandemics.
+    </p>
+    <p>The dashboard provides estimates of:</p>
+    <ul>
+      <li>Prevalence</li>
+      <li>Hospital demand</li>
+      <li>Deaths</li>
```

Comment:
Since updates to time series, which you can see in the new version of the metadata.json file now on main, these now need to be drawn from appStore.metadata?.results.time_series_groups instead of appStore.metadata?.results.time_series